src/js/components/Data/filter.js

```diff
@@ -60,10 +60,16 @@ export const filter = (data, view, properties) => {
           );
 
         // options case
-        if (Array.isArray(filterValue))
-          return Array.isArray(value)
-            ? !value.some((v) => filterValue.includes(v))
-            : !filterValue.includes(value);
+        if (Array.isArray(filterValue)) {
+          if (Array.isArray(value)) {
+            return !filterValue.some((f) =>
+              typeof f === 'object'
+                ? value.includes(f.value) // from {label: ___, value: ___}
```

Comment:
Can we change this to be `f?.value`? Just so if the object doesn't have a value it doesn't cause the app to crash
```suggestion
                ? value.includes(f?.value) // from {label: ___, value: ___}
```